src/Core/Checkout/Order/Aggregate/OrderDelivery/OrderDeliveryEntity.php

```diff
@@ -30,9 +30,9 @@ class OrderDeliveryEntity extends Entity
     protected $shippingMethodId;
 
     /**
-     * @var string|null
+     * @var string[]
```

Comment:
I'm not sure about this change in general, because in shopware 6 an order can have multiple deliveries and wouldn't it be the way to go to split the order in multiple deliveries with their own tracking codes?
But not sure about your concrete use case here.

Reply:
I think it's about shipping with via multiple companies. There are cases when the forwarder changes, even within the country. That could end up in multiple tracking numbers. Asian countries can do that more often than european countries.

Reply:
@keulinho @marcelbrode please see the internal slack thread :wink: 
in short: the package content is too big for one, so it's getting split up. same delivery but multiple packages. 
Re-sending the package if the first address was wrong. same delivery but an additional code.
Retoure packages belonging to the same delivery

Reply:
Got it, makes sense now